src/main/scala/com/normation/cfclerk/domain/AixPasswordHashAlgo.scala

```diff
@@ -231,7 +231,7 @@ object AixPasswordHashAlgo extends Loggable {
       // was introduced in Java 8.
       // Fallback to ssha1 hash.
       logger.error("Your current Java installation does not support PBKDF2WithHmacSHA512 algorithm, " +
-          "which is necessary for {ssha256} hash. Falling back to {ssha1} hashing scheme")
+          "which is necessary for {ssha512} hash. Falling back to {ssha1} hashing scheme")
```

Comment:
If ssha512 is not supported, ssha256 is not supported too (it's written a few lines above)